### ISSUE TITLE
Video Demo Missing SMPL File

### ISSUE THREAD

**Issue by snitchjinx** [saidwivedi/TokenHMR#8]
Hi! Thank you for sharing this work!

When I tried to run the demo on video, I encountered the following error message:

`Error executing job with overrides: ['video.source=demo_sample/video/gymnasts.mp4', 'render.colors=slahmr', '+checkpoint=data/checkpoints/tokenhmr_model.ckpt', '+model_config=data/checkpoints/model_config.yaml']
Traceback (most recent call last):
  File "tokenhmr/track.py", line 68, in main
    phalp_tracker = PHALP_Prime_TokenHMR(cfg)
  File "tokenhmr/track.py", line 50, in __init__
    super().__init__(cfg)
  File "/home/liujy/Documents/TokenHMR/venv/lib/python3.8/site-packages/phalp/trackers/PHALP.py", line 50, in __init__
    self.cached_download_from_drive()
  File "/home/liujy/Documents/TokenHMR/venv/lib/python3.8/site-packages/phalp/trackers/PHALP.py", line 659, in cached_download_from_drive
    convert_pkl('basicModel_neutral_lbs_10_207_0_v1.0.0.pkl')
  File "/home/liujy/Documents/TokenHMR/venv/lib/python3.8/site-packages/phalp/utils/utils.py", line 136, in convert_pkl
    with open(old_pkl, "rb") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'basicModel_neutral_lbs_10_207_0_v1.0.0.pkl'
`
I have downloaded data as instructed. I wonder did I miss any setup step?

Thank you for your consideration!

**Comment by saidwivedi** [comment]
Thank you for trying out the demo. The error is due to missing SMPL file PHALP codebase. I have updated the forked version (see [here](https://github.com/saidwivedi/PHALP/commit/c3cf0e37189fde554acacf4d2864e2985b241a5f). I have updated the TokenHMR readme for new instructions.

P.S: I have also released a new TokenHMR model (see the README) which has better image alignment than the original model. Feel free to test that.
